### PR TITLE
refactor(refs T30969): move DpButtonRow to components root

### PR DESCRIPTION
**Pull request by hwiem**
Moves DpButtonRow from the `core` directory to the `components` directory (so we can remove the `core` directory later).

Ticket: https://yaits.demos-deutschland.de/T30969

### CODE REVIEW

File: src/components/core/DpEditor/DpLinkModal.vue [hunk 1]
```diff
@@ -46,7 +46,7 @@
 </template>
 
 <script>
-import DpButtonRow from '../DpButtonRow'
+import DpButtonRow from '../../DpButtonRow/DpButtonRow'
```
Comment (spiess-demos) [inline src/components/core/DpEditor/DpLinkModal.vue:49]: The longer i look at those paths, the more i want to [use aliases to access other modules](https://webpack.js.org/configuration/resolve/#resolvealias). However i guess this can be a completely separate PR.

Comment (hwiem) [reply src/components/core/DpEditor/DpLinkModal.vue:49]: I was wondering about the paths, too. We could also import from the index file, would you consider that a better solution?
